deps/rabbitmq_management/test/rabbit_mgmt_http_SUITE.erl

```diff
@@ -1878,6 +1878,22 @@ defs_default_queue_type_vhost(Config, QueueType) ->
 
     %% Create a test vhost
     http_put(Config, "/vhosts/test-vhost", #{default_queue_type => QueueType}, {group, '2xx'}),
+
+    Args = #{name => <<"test-vhost">>,
```

Comment:
Are you sure this really belongs to `defs_default_queue_type_vhost/2`? I personally would create a new test.